pytest_tests/testsuites/object/test_object_lock.py

```diff
@@ -684,3 +752,45 @@ def test_link_object_of_complex_object_should_also_be_protected_from_deletion(
                     self.shell,
                     self.cluster.default_rpc_endpoint,
                 )
+
+    @allure.title("Locked object should be protected from deletion after the storage nodes are restarted")
+    @pytest.mark.parametrize(
+        # Only complex objects are required for this test
```

Comment:
why?

Reply:
> why?

Is it important to check the simple object here as well? If you think it's important, I'll add simple_object_size too.

Reply:
> the logic behind the complex objects are complicated enought to treat them an absolutelly different way of object storage, so i would expect testing both always. i have fixed both the bugs that relate the simple objects only and the bugs that affects chains of the objects

Fixed